### ISSUE TITLE
bug when passing a string to the input field

### ISSUE THREAD

**Issue by z1rt** [bugy/script-server#652]
![Снимок](https://user-images.githubusercontent.com/48543053/232360478-66799392-7946-418f-b5cf-de2087a36d05.PNG)

I need to pass any string to after for input, but the field does not accept the special characters " ', " , - as an example.

**Comment by bugy** [comment]
Hi @z1rt I assume this is powershell? Could it be, that you are passing parameters to other scripts/programs without quoting them?
Script server doesn't do any preprocessing and passes values as it is.

**Comment by z1rt** [comment]
Yes, I pass parameters from the input field to the powershell script. Can you tell me how to process the string in this case so that there is no error?

**Comment by bugy** [comment]
I don't know much about powershell and your script. But one of the first things to check is that you always wrap all the parameters in quotes inside the script, e.g. 
Bad
`echo $myField1`
Good
`echo "$myField"`

Sorry if it sounds obvious, but this is the most common issue I saw.

**Comment by z1rt** [comment]
The problem is not in the powershell script, but in the fact that the input field on the interface of your script server does not perceive some characters. That is, it perceives ordinary strings easily, until there is a quotation mark or a comma, for example. 

The main idea of my script is to perform authorization in a powershell script, through your script interface. But I can't pass the password string, as an error occurs from the attachment.

At first I also thought about processing variables in the script itself, but it turned out that they do not pass through the interface

Do I need to do input processing in the interface?

**Comment by bugy** [comment]
Is it just a normal input field? Could you check in "history" section, how
does a command look like?

z1rt ***@***.***> schrieb am Mo., 17. Apr. 2023, 12:25:

**Comment by z1rt** [comment]
Yes, it's just a text field with the type "text"

**Comment by z1rt** [comment]
![image](https://user-images.githubusercontent.com/48543053/232465032-c2907c24-e039-4554-80c0-82dc43938cfd.png)

**Comment by z1rt** [comment]
Wow, thank you very much!
